### DISCUSSION TITLE
Is there some words missing from this paragraph in the documentation?

### DISCUSSION THREAD

**Discussion by Richardinho** [babel/babel#12305] [General] [answered]
From this page in the documentation: https://babeljs.io/docs/en/babel-preset-env#debug

> NOTE: When using core-js@2 (either explicitly using the corejs: 2 option or implicitly), _@babel/preset-env will also imports and requires of @babel/polyfill_. This behavior is deprecated because it isn't possible to use @babel/polyfill with different core-js versions.

The  part in italics doesn't seem to make sense. It looks like either *imports* and *requires* are spelled wrongly (plural nouns apparently being used as verbs), or else there are some missing words. Unfortunately it makes it very hard to understand.

**Comment by nicolo-ribaudo** [comment, marked as answer]
Yes, thanks!

It should probably be something like

> NOTE: When using core-js@2 (either explicitly using the corejs: 2 option or implicitly), @babel/preset-env will also **transform** `import "@babel/polyfill"` and `require("@babel/polyfill")`. This behavior is deprecated because it isn't possible to use @babel/polyfill with different core-js versions.

Would you like to open a PR at https://github.com/babel/website fixing it?

**Reply by Richardinho** [reply]
Sure, I'll give it a go.
